11-Backbonejs/events.md

```diff
@@ -0,0 +1,207 @@
+# Working with Backbone Events
```

Comment:
It seems like this could potentially be two different resources. One that explains how events work overall (with the vocab and diagrams) and then a second with the implementation examples and exercises. Thoughts?

Reply:
I think the theory one would not be big enough to justify a whole separate resource.